index.html

```diff
@@ -0,0 +1,121 @@
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/Linkedin-icon.png"
+                                    alt="not found"></a></li>
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluM.png"
+                                    alt="not found"></a></li>
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluGit.png"
+                                    alt="not found"></a></li>
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluVector.png"
+                                    alt="not found"></a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+        <section class="aboutme">
+            <div class="card-works">
+                <h1>About Myself</h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through
+                    some of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.
+                </p>
+                <b>LET'S CONNECT</b>
+                <div class="project-info">
+                     <ul>
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/bluTwitter.png"
+                                    alt="not found"></a></li>
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/Linkedin-icon.png"
+                                    alt="not found"></a></li>
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluM.png"
+                                    alt="not found"></a></li>
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluGit.png"
+                                    alt="not found"></a></li>
+                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluVector.png"
+                                    alt="not found"></a></li>
+                    </ul>
+                    <p><button>Get my resume</button></p>
+                </div>
+                <h3>Lanuages</h3>
+                <ul class="languages">
+                    <li>
+                        <div class="html-lang-img"></div>
+                        <h5>html</h5>
+                    </li>
+                    <li>
+                        <div class="css-lang-img"></div>
+                        <h5>css</h5>
+                    </li>
+                    <li>
+                        <div class="javascript-lang-img"></div>
+                        <h5>javascript</h5>
+                    </li>
+                </ul>
+                <button>frameworks <span class="fa fa-chevron-right"></span></button>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a  ul, each of them in a li. This is a project requirement 🤝 
```
 <ul>
    <li>Languages</li>
    <li>Framework</li>
    ...
</ul>
```